### ISSUE TITLE
Error evaluating code

### ISSUE THREAD

**Issue by igorkulman** [enricosada/LightTable-FSharp#1]
On Windows 8.1, with FSI in the path, I get the following error trying to evaluate anything:

```
at Microsoft.FSharp.Compiler.Interactive.Shell.FsiInteractionProcessor.commitResult(FsiDynamicCompilerState istate, FsiInteractionStepStatus result) in C:\GitHub\fsharp\FSharp.Compiler.Service\src\fsharp\fsi\fsi.fs:line 1946
   at Microsoft.FSharp.Compiler.Interactive.Shell.FsiInteractionProcessor.EvalExpression(String sourceText) in C:\GitHub\fsharp\FSharp.Compiler.Service\src\fsharp\fsi\fsi.fs:line 2076
   at FSI_0001.eval(String text) in C:\Users\igor\AppData\Local\LightTable\plugins\F#\fs-src\ltfsclient\ltfsclient.fsx:line 69
   at FSI_0001.handle(Stream stream, Request request) in C:\Users\igor\AppData\Local\LightTable\plugins\F#\fs-src\ltfsclient\ltfsclient.fsx:line 169
Evaluation failed
```

Maybe a hardocded path to C:\GitHub\fsharp?

**Comment by enricosada** [comment]
can i have source file and what are you selecting?

atm work only with selection like
1+3
System.DateTime.Now
[0..2..10]

but not
let add x y = x + y

the path you see is FSharp.Compiler.Service debug symbols, where was compiled

**Comment by igorkulman** [comment]
I was trying let x=5;; so I guess it is not supported either.

**Comment by enricosada** [comment]
binding next version, only evaluation of values now

**Comment by enricosada** [comment]
fixed with version 0.1.1

`let add x y = x + y` **✓**

`add 1 3` **4**

`add` **fun:it@1**

`let add x y = x + y + 3` **✓**

`add 1 3` **7**

`add` **fun:it@1-1**
